apps/client/src/environments/patch-notes.ts

```diff
@@ -1,16 +1,16 @@
-export const patchNotes = `### Bug Fixes
-
-* **community-lists:** fixed community lists content not showing.
-* **list:** fixed an issue with ephemeral lists not being created properly.
-* **list:** fixed zone breakdown rows not showing item name.
-* **list-overlay:** fixed list panel overlay not using selected layout.
-* **profile:** fixed an issue with profile not loading properly.
-* **recipe-picker:** fixed result names always showing in english.
-* **rotations:** fixed loading screen not showing properly.
-* **simulator:** fixed a bug with simulator not opening properly.
-* **teams:** fixed team detach from lists page.
-
-
-### Features
-
-* **patreon:** new supporter: Coss Collaborative..`;
+export const patchNotes = `### Bug Fixes
+
+* **community-lists:** fixed community lists content not showing.
+* **list:** fixed an issue with ephemeral lists not being created properly.
+* **list:** fixed zone breakdown rows not showing item name.
+* **list-overlay:** fixed list panel overlay not using selected layout.
+* **profile:** fixed an issue with profile not loading properly.
+* **recipe-picker:** fixed result names always showing in english.
+* **rotations:** fixed loading screen not showing properly.
+* **simulator:** fixed a bug with simulator not opening properly.
+* **teams:** fixed team detach from lists page.
+
+
+### Features
+
+* **patreon:** new supporter: Coss Collaborative..
`;
```

Comment:
Idk what you added there but it's not needed ^^

Reply:
Well Idk too, maybe did a backspace without paying attention ^^"